packages/bolt-connection/src/connection/connection-channel.js

```diff
@@ -331,6 +335,16 @@ export default class ChannelConnection extends Connection {
               if (telemetryEnabledHint === true) {
                 this._telemetryDisabledConnection = false
               }
+
+              const SSREnabledHint = metadata.hints['ssr.enabled']
+              if (SSREnabledHint) {
```

Comment:
```suggestion
              if (SSREnabledHint === true) {
```
Realistically, we should never see any other truthy value from the server, but I'm in favor of practicing defensive input validation :)